docs/csharp/language-reference/configure-language-version.md

```diff
@@ -36,6 +36,9 @@ The compiler determines a default based on these rules:
 
 When your project targets a preview framework that has a corresponding preview language version, the language version used is the preview language version. You use the latest features with that preview in any environment, without affecting projects that target a released .NET Core version.
 
+> [!IMPORTANT]
```

Comment:
We should merge this with the next alert or alter the positioning. I think it's bad UX to have back-to-back alerts.

Reply:
I moved the tip into the next section